content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+view of {{< gloss >}}service endpoint{{< /gloss >}}s. The view depends on the
+network of the consumers.
+
+{{< image width="75%"
+    link="./multi-net.svg"
+    alt="A service mesh with multiple networks"
+    title="Multi-network deployment"
+    caption="A service mesh with multiple networks"
+    >}}
+
+## Mesh models
+
+Istio supports two mesh models:
+
+- Single mesh
+- Multiple meshes, also known as **multi-mesh**
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. Within a mesh, service names are
+unique. For example, only one service can have the name `foo/mysvc`.
+Additionally, workload instances all share a common identity and authentication
+infrastructure and can authenticate each other. With a single mesh you can do
+the following tasks:
+
+- Span [one or more clusters](#cluster-models)
+- Span [one or more networks](#network-models)
+- Use [namespaces for tenancy](#namespace-tenancy)
+
+### Multiple Meshes
+
+You can deploy multiple meshes; such a configuration is known as
+**multi-mesh**.
+
+Multiple meshes afford the following capabilities beyond that of a single mesh:
+
+- Organizational boundaries: lines of business
+- Service name or namespace collisions: multiple distinct uses of the `default`
+  namespace
+- Stronger isolation: isolating test workloads from production workloads
+
+In the most basic multiple mesh model, each mesh forms a strict
+boundary and there is no communication between meshes. However, you can enable
+inter-mesh communication with **mesh peering**. When
+peering, each mesh can export a set of services and identities, which all
+participating meshes can recognize.
+
+{{< image width="75%"
+    link="./multi-mesh.svg"
+    alt="Multiple service meshes"
+    title="Multi-mesh"
+    caption="Multiple service meshes"
+    >}}
+
+To avoid service naming collisions, you can give each mesh a globally unique
+**mesh ID**, to ensure that the fully qualified domain
+name (FQDN) for each service is distinct.
+
+When peering two meshes that do not share the same
+{{< gloss >}}trust domain{{< /gloss >}}, {{< gloss >}}identity{{< /gloss >}} and
+**trust bundles** must be federated between them. See the section
+on [Multiple Trust Domains](#trust-between-meshes) for an overview.
+
+## Control plane models
+
+The Istio [control plane](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-control-plane)
+configures the mesh or a subset of the mesh to manage the communication between
+workload instances. A single deployment of the control plane consists of several
+redundant processes that share a common view of the mesh and provide workload
+instances with identical configuration.
+
+Depending on your overall system architecture and cloud provider, the following
+Istio control plane models are possible:
+
+- [single control plane](#single-control-plane)
+- [multiple control planes](#multiple-control-planes)
+- [managed control planes](#managed-control-plane)
+
+### Single control plane
```

Comment:
As @costinm has pointed out several times, a single control plane is not a realistic deployment. The topology that we're interested in describing is a "shared" control plane, i.e., a control plane that is shared by more than one cluster. Even in this case, it would be likely that we would have multiple shared control planes for HA.

Reply:
> Also note that this section is specifically about single cluster meshes, in which case a single control plane is fine. But of course that is also a control plane per cluster...

That's right ... with a single cluster, you would only have a single control plane deployment. And that's what this section calls out. 

> Ideally we can describe that you can have one or more logical control planes, each of which is typically replicated into one or more clusters. Some extremes are control plane per cluster, or single logical control plane with multiple replicas (which is very similar to a managed control plane, just a self-managed one).

@smawson let's chat today so I have a better understanding of the various control plane models you're referring to. I'd like to condense this down to something that's reasonably abstract here.

Reply:
I agree. How about we start by changing this section to "Shared control plane" and change the diagram to be similar to the multiple control planes diagram, bit with a control plane in only one of the clusters.

We should also talk about how HA would be configured with this shared control plane setup.

Reply:
@nmittler
> with a single cluster, you would only have a single control plane deployment

Please note that you can have multiple control planes per single cluster, see this blog post https://istio.io/blog/2018/soft-multitenancy/

Reply:
@vadimeisenbergibm do you think we can skip going into that level of detail? Technically you could have a single cluster with many control planes, each in a different mesh :) Maybe we can just point at that blog and indicate that istio actually supports a wide range of control plane setups, but only a few standard ones?

Reply:
@smawson I did not mean to add the multiple control planes in a single cluster to the document, just to make sure @nmittler is aware of that pattern.